### ISSUE TITLE
Indicator not showing up in Ubuntu 15.10

### ISSUE THREAD

**Issue by metalbrick** [vikoadi/indicator-kdeconnect#31]
the KDE-Connect Worked perfectly, but there's no indicator in the panel, thus I can no longer use the features such like "Browse device" conveniently.

**Comment by Deus-Rex** [comment]
Having same problem.

If I try to start the indicator using terminal, I get the following:

`** Message: KDEConnectManager.vala:32: KDE Connect daemon found
** Message: Device.vala:73: device : /modules/kdeconnect/devices/1b34fcf462e1729
** Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/1b34fcf462e1729' (signature '')
** Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/1b34fcf462e1729' (signature '')
** Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/1b34fcf462e1729' (signature '')
** Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/1b34fcf462e1729' (signature '')`

**Comment by Manuel-Kehl** [comment]
I'm getting the same list of error messages like @Deus-Rex when trying to start it on Arch.

**Comment by kill1n** [comment]
Same problem here.

OS: Ubuntu 15.10 (unity)

KDE Connect works fine. Ping, notifications, remote mouse...

indicator-kdeconnect doen't show the tray icon. Sometimes show this window, other times don't show anything.

Terminal output:

user@ubuntu15.10:~$ indicator-kdeconnect 
*\* Message: KDEConnectManager.vala:32: KDE Connect daemon found
*\* Message: Device.vala:73: device : /modules/kdeconnect/devices/403a4015353e5b12
*\* Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/403a4015353e5b12' (signature '')
*\* Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/403a4015353e5b12' (signature '')
*\* Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/403a4015353e5b12' (signature '')
*\* Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/403a4015353e5b12' (signature '')

![seleccion_001](https://cloud.githubusercontent.com/assets/16135412/11561674/3671e47e-99c9-11e5-8bcb-b12dab55679d.png)

**Comment by ProtoJazz** [comment]
Im having the same issue with a new arch install. No icon. Everything else works fine. I get messages, can even remote mouse. But no icon with Arch + XFCE 4

Any ideas?

**Comment by ghost** [comment]
Same problem here: OS: Ubuntu 15.10 (unity), no tray icon.

**Comment by FloEdelmann** [comment]
I know that I did have the same problem with the ownCloud desktop sync indicator until I changed the apt-repository and got the updated application from there. Maybe that helps finding the cause.

**Comment by varlesh** [comment]
I build packages for Wily. Please use this repository:
https://launchpad.net/~varlesh-l/+archive/ubuntu/ubuntu-tools
- Delete kdeconnect package and dependecies:

```
sudo apt-get purge kdeconnect
sudo apt-get autoremove
```
- Add repository:

```
sudo add-apt-repository ppa:varlesh-l/ubuntu-tools
sudo apt-get update
```
- Open Synaptic, search "kdeconnect" - Install kdeconnect & indicator-kdeconnect packages on Synaptic (select new version "kdeconnect_0.8.1wily2" on Menu-Package)
  Indicator not working because on official Wily repository used kde5-libs, but indicator working only kde4-libs.

**Comment by kill1n** [comment]
Hi and thanks varlesh.

I'm using your instructions. Now i can see the indicator and seems work the most options. Only, when i click "browse the device", nautilus open a blank folder.

Other options like battery percentage, ping, mouse, notifications, etc.. seems work fine.

I never use kdeconnect previously. I forgot something?

![seleccion_002](https://cloud.githubusercontent.com/assets/16135412/11912263/a20460ea-a635-11e5-8724-a3c45952c2d2.png)

**Comment by khartahk** [comment]
I'm also not seeing the icon on gnome shell 3.18 on arch. When I installed systemsettings package for KDE I could pair the device and message notifications work, I can send files from the phone but not to it. I can control the mouse, keyboard on the computer. The only thing missing is browse the phone and send to phone from nemo and the indicator icon. I tried to install the nautilus-kdeconnect python script but it's not working with nemo. Any direction on what to do next will be appreciated.

**Comment by watermelonjuice** [comment]
Any update on this?

**Comment by sergioad** [comment]
Hello @varlesh I am also having issues with KDE connect, I am in Xubuntu 16.04 and kDE connect indicator does not appears, if you have any solution or you know a workarround for this please let me know, greetings from Mexico :smile: 

PS: I recommend you to upgrade it to work with Qt5 :smile:

**Comment by tristan-k** [comment]
@sergioad Isnt this related to XFCE? Have a look at http://blog.alteroot.org/articles/2013-11-16/kdeconnect-with-android-and-xfce-archlinux.html

**Comment by tristan-k** [comment]
I have to correct myself here. @rcristi  [fork](https://github.com/rcristi/indicator-kdeconnect) solves the problem. Here are the steps to compile on Xubuntu 16.04:

```
sudo apt-get install gtk+-3.0-dev libappindicator3-dev valac
git clone https://github.com/rcristi/indicator-kdeconnect
cd indicator-kdeconnect
mkdir build
cd build
cmake .. -DCMAKE_INSTALL_PREFIX=/usr
make
sudo make install
```

Here's a binary I built with `checkinstall`

[indicator-kdeconnect_20160512-1_amd64.zip](https://github.com/vikoadi/indicator-kdeconnect/files/261580/indicator-kdeconnect_20160512-1_amd64.zip)

**Comment by sergioad** [comment]
As @vikoadi said before this is bgecause of Qt5 @tristan-k (or at east this is what I think) anyway I will try it soon with the fork you said :smile:

**Comment by lutostag** [comment]
With a new build from https://github.com/rcristi/indicator-kdeconnect I got it working for me. I was running into the same issue.
PPA here: https://launchpad.net/~lutostag/+archive/ubuntu/indicator-kdeconnect-test

(couldn't get it to build for precise with the versioned dependencies). Not sure if they are necessary of not.

Still can't get remote file browsing to work though.

**Comment by bam80** [comment]
@lutostag , still can't get indicator visible in Ubuntu 15.10 even with your PPA.
indicator-kdeconnect prints nothing wile running from terminal

**Comment by lutostag** [comment]
@bam80 yeah unfortunately it is still a bit finicky. The only way I get it to work currently is to run:
kcmshell5 kcm_kdeconnect # make sure it it connected to your device, then only after that...
indicator-kdeconnect #works for me and shows up in the systemtray.

**Comment by ghost** [comment]
@vikoadi @bam80 using @tristan-k 's .deb and running @lutostag 's commands as a startup script, i have managed to reliably get the indicator to appear in ubuntu 16.04, only if my screen is turned on

[kdeConnect.txt](https://github.com/vikoadi/indicator-kdeconnect/files/520441/kdeConnect.txt)

**Comment by markcross** [comment]
LXLE 16.04.1 got indicator-kdeconnect working of sorts with this, but no nice icon in the applet-indicators thingy at the top of LXLE

https://github.com/rcristi/indicator-kdeconnect
PPA here: https://launchpad.net/~lutostag/+archive/ubuntu/indicator-kdeconnect-test

**Comment by varlesh** [comment]
@markcross New version here:
https://github.com/Bajoja/indicator-kdeconnect
